### ISSUE TITLE
Unable to preventDefault inside passive event listener

### ISSUE THREAD

**Issue by ajuanjojjj** [soundar24/roundSlider#98]
Hello,

I'm working on a cordova app wich implements your roundslider, and even though it appears to work just fine, the console/logcat gets filled with this [Intervention] error, one for each pixel i move the control. 
I'm not exactly sure if this is something directly caused by roundSlider or more of a jquery problem, and if there are any quick fixes to it

The error itself is the following: 

[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/5093566007214080
```
preventDefault @ jquery.js:formatted:2053
_handleMove @ roundslider.min.js:formatted:444
f @ jquery.js:formatted:192
dispatch @ jquery.js:formatted:1931
r.handle @ jquery.js:formatted:1858
```
Thanks in advance

**Comment by soundar24** [comment]
@ajuanjojjj yes I can reproduce this issue, and this is due to roundSlider only. By applying the below style you can resolve this issue.

```
.rs-handle {
    touch-action: none;
}
```

Please confirm me whether this solved the issue or not, so that I will integrate this with the source.

**Comment by ajuanjojjj** [comment]
Sorry for the delayed response, I was quite busy yesterday and forgot to try your solution

Just added it to the CSS and worked like a charm, thank you very much!

**Comment by soundar24** [comment]
@ajuanjojjj this fix was added in the source level and it's now available in the [roundSlider v1.5.1](https://github.com/soundar24/roundSlider/releases/tag/v1.5.1), so you can use this version. Thanks...

**Note:** Also from the recent versions you can use the **[svgMode](https://roundsliderui.com/document.html#svgMode)**, which has lots of improvements.
